x-pack/plugins/threat_intelligence/public/modules/indicators/components/query_bar/query_bar.stories.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { __IntlProvider as IntlProvider } from '@kbn/i18n-react';
+import { Storage } from '@kbn/kibana-utils-plugin/public';
+import { ComponentStory } from '@storybook/react';
+import { KibanaContextProvider } from '../../../../hooks/use_kibana';
+import { QueryBar } from './query_bar';
+
+export default {
+  component: QueryBar,
+  title: 'QueryBar',
+  argTypes: {
+    onSubmitQuery: { action: 'onSubmitQuery' },
+    onChangedQuery: { action: 'onChangedQuery' },
+    onChangedDateRange: { action: 'onChangedDateRange' },
+    onSubmitDateRange: { action: 'onSubmitDateRange' },
+    onSavedQuery: { action: 'onSavedQuery' },
+    onRefresh: { action: 'onRefresh' },
+  },
+};
+
+const services = {
+  data: { query: {} },
+  storage: new Storage(localStorage),
+  uiSettings: { get: () => {} },
+};
+
+const Template: ComponentStory<typeof QueryBar> = (args) => (
+  <IntlProvider>
+    <KibanaContextProvider services={services}>
+      <QueryBar {...args} />{' '}
+    </KibanaContextProvider>
+  </IntlProvider>
+);
+
+export const Basic = Template.bind({});
```

Comment:
this fails to load for me
![Screen Shot 2022-08-01 at 9 16 11 AM](https://user-images.githubusercontent.com/17276605/182093630-3a2540bf-d923-46ff-bc1f-0fc70417cc1c.png)

Reply:
ignore this, it works I must have been in a weird local state!